templates/demo-store/app/components/SortFilter.tsx

```diff
@@ -129,32 +118,29 @@ export function FiltersDrawer({
           Filter By
         </Heading>
         <div className="divide-y">
-          {filters.map(
-            (filter: Filter) =>
-              filter.values.length > 1 && (
```

Comment:
Key change here is removing this constraint on filter.values.length. This was preventing the price filter from being shown. Alternative was adding an exception for price filter, but I think just removing this constraint makes more sense.